pkg/scheduling/requirement.go

```diff
@@ -36,8 +36,10 @@ type Requirement struct {
 	values      sets.Set[string]
 	greaterThan *int
 	lessThan    *int
+	MinValues   *int
 }
 
+// TODO: Make changes to accept MinValues from API
```

Comment:
nit: Should we drop this TODO before we plan on merging this change?

Reply:
This is actually removed in the API changes PR. So, once this PR and the API PR is approved, this will automatically go away. And I plan to merge these 2 PRs together so that we do not block other releases.